### PR TITLE
Add Shrinkwrap

### PR DESCRIPTION
**Pull request by matuskosut**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by matuskosut** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/shrinkwrap/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  -DBUILD_SHARED_LIBS:BOOL=ON \
+  -DCMAKE_PREFIX_PATH:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_INCLUDEDIR=include \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DCMAKE_BUILD_TYPE=Release \
+  ${SRC_DIR}
+
+make -j${CPU_COUNT}
+make install
```
Comment (ocefpaf) [inline recipes/shrinkwrap/build.sh:16]: Do we have a `make check` that we can run?

Comment (matuskosut) [reply recipes/shrinkwrap/build.sh:16]: @ocefpaf not exactly `make check`, but generated makefile actually provides `make test`. Do you think it's okay to add in this place?

Comment (ocefpaf) [reply recipes/shrinkwrap/build.sh:16]: Yep. `make test` works :smile:

Comment (matuskosut) [reply recipes/shrinkwrap/build.sh:16]: @ocefpaf thanks, added make test and passed.

File: recipes/shrinkwrap/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+make test
```
Comment (ocefpaf) [inline recipes/shrinkwrap/build.sh:17]: Do we have that for Windows as well? (Sorry I did not mention that in the first message.)

**Comment by matuskosut** [comment]
@ocefpaf done. I actually skip windows build, because I could't make it work, but I added it not to forget about it..

**Pull request merged**